target/cheri-common/cheri-compressed-cap/cheri_compressed_cap_common.h

```diff
@@ -147,8 +142,8 @@ struct _cc_N(cap) {
 typedef struct _cc_N(cap) _cc_N(cap_t);
 #define _cc_cap_t _cc_N(cap_t)
 
-static inline bool _cc_N(exactly_equal)(struct _cc_N(cap) const* a, struct _cc_N(cap) const* b) {
-    return a->cr_tag == b->cr_tag && a->_cr_cursor == b->_cr_cursor && a->cached_pesbt == b->cached_pesbt;
+static inline bool _cc_N(exactly_equal)(const _cc_cap_t* a, const _cc_cap_t* b) {
```

Comment:
While 256 might be dead, it's possible we'll have 64 one day; maybe worth keeping the `N`?

Reply:
That should be the same due to the `#define _cc_cap_t _cc_N(cap_t)` above?

Reply:
Whoop, yep.

Reply:
(and we have CHERI-64 support these days)